libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/ActivityChecker.cs

```diff
@@ -0,0 +1,372 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AdaptiveCards;
+using Microsoft.Bot.Builder.LanguageGeneration;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
```

Comment:
Usings are in incorrect order. 

Are the Stylecop and FxCop rules turned on? They should not be passing given this.

Reply:
Stylecop is on, but I do not see any warnings/errors. So what is the right order of these usings?